docs/guides/getting-started/setup/integrate.mdx

```diff
@@ -0,0 +1,113 @@
+---
+description: If you already have an existing web project, this guide will walk you through integrating Tauri into your project
+pagination_next: guides/development/development-cycle
+sidebar_position: 3
+sidebar_custom_props:
+  doc_card_image: /img/guides/getting-started/setup/html5-light.svg
+  doc_card_image_dark: /img/guides/getting-started/setup/html5-dark.svg
+---
+
+import Commands from './_fragments/_commands.mdx'
+import TauriInit from './_fragments/_tauri-init.mdx'
+import { InstallTauriCli, InstallTauriApi } from '@theme/Command'
+
+# Integrate into Existing Project
+
+If you already have an existing web project, this guide will walk you through integrating Tauri into your project, whether it is Node.js-based (like [Svelte], [React], [Vue], or [Angular]) or Rust-based (like [Yew] or [Dominator]).
+
+:::info
+
+Before we continue, make sure you have completed the [prerequisites] to have a working development environment.
+
+:::
+
+Although Tauri is compatiable with nearly any frontend framework, we'll use examples with a [React] project throughout this guide. We'll assuming you're starting off with a project structure similar to this:
+
+```
+.
+│── package.json
+│── public
+│   ╰── index.html
+╰── src
+    │── App.css
+    │── App.js
+    │── index.css
+    ╰── index.js
+```
+
+## Create the Rust Project
+
+<TauriInit
+  destDirExplination='Use the path where your compiled web assets will be located. This may be something like "../dist" or "../build".'
+  devPathExplination='This is the path or address your frontend is served at during development. If servering from localhost this could be "https://localhost:3000". This can also be a directory similar to the above question.'
```

Comment:
this needs to be more decisive, no maybes, no ors.
```suggestion
  destDirExplination='use `../build` which is the path where the compiled web assets will be located.'
  devPathExplination='use `http://localhost:300` which is the url for the frontend dev server'
```

Reply:
I tried to make this one a bit less straightforward to force the reader to use a bit more critical thinking. I feel like if we just say "use xyz" then when it doesn't work outside of a React project people won't stop to think why. @FabianLars any thoughts on this? I think we've had a few issues where people just didn't really stop and think about these values and put in an issue because of it.

Reply:
Yeah, basically the same topic as here https://github.com/tauri-apps/tauri-docs/pull/834#discussion_r963604929
It must be really really obvious that these values are specific to the CRA example and may differ depending on the frontend tech.
So maybe something that explains it and/or lists common alternatives and also makes it clear that **in this case** it's ../build and localhost:3000

Reply:
Just fixed that and added some explanation text for the beforexcommands